listenbrainz/webserver/static/js/src/follow/SimilarUsersModal.tsx

```diff
@@ -25,22 +25,33 @@ const SimilarUsersModal = (props: SimilarUsersModalProps) => {
 
   return (
     <>
-      <h3 className="text-center">
-        Users similar to {user.name === currentUser?.name ? "you" : user.name}
+      <h3 className="text-center" style={{ marginTop: "10px" }}>
+        Similar Users
       </h3>
-      <div className="similar-users-list">
-        {similarUsersList.map((listEntry: SimilarUser) => {
-          return (
-            <UserListModalEntry
-              mode="similar-users"
-              key={listEntry.name}
-              user={listEntry}
-              loggedInUserFollowsUser={loggedInUserFollowsUser(listEntry)}
-              updateFollowingList={updateFollowingList}
-            />
-          );
-        })}
-      </div>
+      {similarUsersList.length === 0 ? (
+        <>
+          <hr style={{ margin: "0px 2em", borderTop: "1px solid #eee" }} />
```

Comment:
Should probably be a style for the `similar-users-empty` class defined in a less file as well

Reply:
Applying the properties to hr element in follow.less earlier took care of this hr as well :)

Reply:
Even better !